[2025-05-30T05:25:37.274+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-30T05:25:37.301+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: read_csv_sum_dag.read_and_sum_csv manual__2025-05-30T05:25:35.973244+00:00 [queued]>
[2025-05-30T05:25:37.310+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: read_csv_sum_dag.read_and_sum_csv manual__2025-05-30T05:25:35.973244+00:00 [queued]>
[2025-05-30T05:25:37.311+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2025-05-30T05:25:37.326+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): read_and_sum_csv> on 2025-05-30 05:25:35.973244+00:00
[2025-05-30T05:25:37.336+0000] {standard_task_runner.py:72} INFO - Started process 486 to run task
[2025-05-30T05:25:37.349+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'read_csv_sum_dag', 'read_and_sum_csv', 'manual__2025-05-30T05:25:35.973244+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/read_csv_sum_dag.py', '--cfg-path', '/tmp/tmp_6eu_xai']
[2025-05-30T05:25:37.356+0000] {standard_task_runner.py:105} INFO - Job 21: Subtask read_and_sum_csv
[2025-05-30T05:25:37.394+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.10/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-30T05:25:37.446+0000] {task_command.py:467} INFO - Running <TaskInstance: read_csv_sum_dag.read_and_sum_csv manual__2025-05-30T05:25:35.973244+00:00 [running]> on host 1232a4f48263
[2025-05-30T05:25:37.528+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='read_csv_sum_dag' AIRFLOW_CTX_TASK_ID='read_and_sum_csv' AIRFLOW_CTX_EXECUTION_DATE='2025-05-30T05:25:35.973244+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-30T05:25:35.973244+00:00'
[2025-05-30T05:25:37.529+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-30T05:25:37.603+0000] {logging_mixin.py:190} INFO - Columns in CSV: ['Unnamed: 0', 'Sumextension', 'EmpCode']
[2025-05-30T05:25:37.604+0000] {logging_mixin.py:190} INFO - Column 'your_column_name' not found in CSV
[2025-05-30T05:25:37.605+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-30T05:25:37.622+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-30T05:25:37.622+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=read_csv_sum_dag, task_id=read_and_sum_csv, run_id=manual__2025-05-30T05:25:35.973244+00:00, execution_date=20250530T052535, start_date=20250530T052537, end_date=20250530T052537
[2025-05-30T05:25:37.642+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2025-05-30T05:25:37.656+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-30T05:25:37.658+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
